### DISCUSSION TITLE
How did you find out about UBA?

### DISCUSSION THREAD

**Discussion by otseng** [eliranwong/UniqueBible#170] [General]
How do you find out about the Unique Bible app?  
What drew your interest in it?

**Comment by otseng** [comment]
I operate the online forum [Debating Christianity & Religion](https://debatingchristianity.com/).  In [a thread](https://debatingchristianity.com/forum/viewtopic.php?p=1027061#p1027061), one of our forum members mentioned UBA.  I was intrigued since it was open sourced and downloaded it.  Once I started playing around with it a bit, I was amazed by its power and potential.  I also was impressed with all the Marvel Bibles.  Now, I pretty much use it every day.

**Comment by onegladiator4u** [comment]
I found out through searching. Don't know which search engine showed me, I think it's Duckduckgo.
I was searching for an app that can show my language bible clearly. I couldn't through esword. 
When I found out this app I didn't install it because it really looked complicated with PowerShell. But app looked awesome and different. 
After many months I remembered again and found it again. It was really worth it after installing an app different way.

**Comment by pvt02** [comment]
I was searching for an alternative to E-sword on Linux and found a link or mention somewhere.

**Comment by Surcouf01** [comment]
I found about uba while chating with Mistral AI on related subject.
